src/vea/vea_internal.h

```diff
@@ -149,6 +173,7 @@ int verify_resrvd_ext(struct vea_resrvd_ext *resrvd);
 int vea_dump(struct vea_space_info *vsi, bool transient);
 int vea_verify_alloc(struct vea_space_info *vsi, bool transient,
 		     uint64_t off, uint32_t cnt);
+void update_statistics(struct vea_space_info *vsi, unsigned int type, uint64_t nr, bool dec);
```

Comment:
Using a bool for whether to add or subtract the number makes it a little harder to recognize what's going on, as a reader. I had a guess while reading the code where it's called, but wasn't sure until I got to the function signature. I'd suggest having an inc_stats and dec_stats instead for clarity.

Reply:
ok.